src/test/java/ca/corefacility/bioinformatics/irida/service/impl/unit/sample/SampleServiceImplTest.java

```diff
@@ -428,7 +433,8 @@ public void testMergeMetadataSuccess() {
 
 		sampleService.mergeSampleMetadata(s1, inputMetadata);
 
-		ArgumentCaptor<Set> saveCaptor = ArgumentCaptor.forClass(Set.class);
+		@SuppressWarnings("unchecked")
```

Comment:
Still required in this file as well?

Reply:
Yes it is required to supress the warnings on the line below.